JS/Lesson35/onder/my-app/src/components/Auth/LogIn.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useContext, useState } from "react";
+import { Link } from "react-router-dom";
+import { Button, TextField, Typography, Grid, Paper } from "@mui/material";
+import {useAuth } from "./AuthProvider";
+
+const LogIn = () => {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  const {isLoggedIn, login, logout} = useAuth();
+
+  const handleLogIn = () => {
+    if(email==="onder" && password==="123"){
+      login();
+    }
+    else{
+      logout();
+    }
+    console.log('button Clicked '+isLoggedIn);
+    
+  };
+
+  return (
+    <Grid container justifyContent="center">
+      <Grid item xs={12} sm={6} md={4} margin={7}>
+        <Paper elevation={3} sx={{ padding: 2 }}>
+          <Typography variant="h5" gutterBottom>
+            Log In
+          </Typography>
+          <TextField
+            fullWidth
+            label="Email"
+            margin="normal"
+            value={email}
+            onChange={(e) => setEmail(e.target.value)}
+          />
+          <TextField
+            fullWidth
+            label="Password"
+            type="password"
+            margin="normal"
+            value={password}
+            onChange={(e) => setPassword(e.target.value)}
+          />
+          <Button variant="contained" onClick={handleLogIn} sx={{ mt: 2 }}>
+            LogIn
+          </Button>
+          
```

Comment:
I don't think anything on lines 49-62 is really needed. Log in form should only be shown to users that are not logged in. If a logged in user navigates to the login page, it should be redirected to, e.g. main page.